webapp/channels/src/components/post_view/channel_intro_message/add_members_button.tsx

```diff
@@ -137,23 +121,22 @@ const MoreThanMaxFreeUsers = ({channel, setHeader, pluginButtons}: {channel: Cha
                         {isPrivate && channel.group_constrained &&
                             <FormattedMessage
                                 id='intro_messages.inviteGropusToChannel.button'
-                                defaultMessage='Add groups to this private channel'
+                                defaultMessage='Add groups'
                             />}
                         {isPrivate && !channel.group_constrained &&
                             <FormattedMessage
-                                id='intro_messages.inviteMembersToPrivateChannel.button'
-                                defaultMessage='Add members to this private channel'
+                                id='intro_messages.inviteMembersToChannel.button'
+                                defaultMessage='Add people'
                             />}
                         {!isPrivate &&
                             <FormattedMessage
                                 id='intro_messages.inviteMembersToChannel.button'
-                                defaultMessage='Add members to this channel'
+                                defaultMessage='Add people'
                             />}
```

Comment:
Since the text is now the same for public and private channels, could you combine these two cases?
```jsx
                        {!channel.group_constrained &&
                            <FormattedMessage
                                id='intro_messages.inviteMembersToChannel.button'
                                defaultMessage='Add people'
                            />}
```

Reply:
Good call. Fixed that now.